.github/workflows/release.yml

```diff
@@ -34,7 +34,8 @@ jobs:
           git checkout main
           git config user.name swellbot
           git config user.email opensource@swellaby.com
-          npm run package:vsix -- patch -m "chore: bump version [skip ci]"
+          npm version patch -m "chore: bump version [skip ci]"
+          npm run package:vsix
```

Comment:
Best I can tell something's changed and that the internal version bumping `vsce` is performing results in `vsce` losing track of the package version from the manifest, which rolls downhill somewhat in how the vsix files are named.

This should get things sorted, albeit in a mildly less succinct manner